File: C/PRAK204-2210817310017-Rohid_Akmal.c
#include <stdio.h>

void main(){
    float r, t, v, l, k, phi = 3.142857;

    printf("Jari-jari  : ");
    scanf("%f", &r);
    printf("Tinggi     : ");
    scanf("%f", &t);

    v = phi * (r*r) * t;
    printf("Volume  : %.2f\n", v);
    l = 2* phi * r * (r + t);
    printf("Luas    : %.2f\n", l);
    k = 2 * phi * r;
    printf("Kelling : %.2f\n", k);
}

File: Python/PRAK204-2210817310017-Rohid Akmal.py
r = int(input("Jari-jari = "))
t = int(input("Tinggi = "))
phi = 3.142857
v = phi * (r*r) * t
l = 2* phi * r * (r + t);
k = 2 * phi * r;
print("Volume = %.2f "%(v))
print("Luas = %.2f "%(l))
print("Keliling = %.2f "%(k))